src/book/services/BooksService.test.tsx

```diff
@@ -1,89 +1,132 @@
 import { renderHook } from "@testing-library/react-hooks";
-import { act } from "react-dom/test-utils";
-import { useBooks } from "./BooksService";
+import { getURI, useBooks } from "./BooksService";
 import { Book } from "../book";
 
-const books = [
+const mockedResponseBooks: Book[] = [
   {
     id: 1,
-    authors: "John Example",
-    title: "Example Book",
+    authors: "Julius Verne",
+    title: "80 days around the world",
   },
   {
     id: 2,
-    authors: "Joe Smith",
-    title: "Another Book",
+    authors: "Frank Herbert",
+    title: "Dune",
   },
 ];
 
+interface HttpRequestConfig {
+  method: "GET" | "POST" | "PUT" | "DELETE";
+  headers: { "Content-Type": string };
+  body: any;
+}
+
```

Comment:
mockFetch for this test suite, we mock every reqest which will be triggered by our service method